instrumentation/kafka/kafka-clients/kafka-clients-0.11/testing/build.gradle.kts

```diff
@@ -10,4 +10,10 @@ dependencies {
   implementation(project(":instrumentation:kafka:kafka-clients:kafka-clients-common:library"))
 
   implementation("org.testcontainers:kafka")
+  implementation("org.testcontainers:junit-jupiter")
+
+  compileOnly("com.google.auto.value:auto-value-annotations")
+  annotationProcessor("com.google.auto.value:auto-value")
+
+  runtimeOnly("org.apache.kafka:kafka_2.13:2.8.1")
```

Comment:
is this dependency needed after switching to testcontainers?

Reply:
No its not. Thanks!